src/Models/Validator.php

```
@@ -87,21 +87,22 @@ protected function validateNumberAboveMinimum($field, $value, $minimum)
      * @throws \GloBee\PaymentApi\Exceptions\Validation\InvalidUrlException
      * @throws \GloBee\PaymentApi\Exceptions\Validation\InvalidArgumentException
      */
-    protected function validateUrl($field, $url)
+    public static function validateUrl($field, $url)
     {
-        $this->validateString($field, $url);
+        self::validateString($field, $url);
         if (filter_var($url, FILTER_VALIDATE_URL) === false) {
             throw new InvalidUrlException($field, $url);
```

lets have this only throw a InvalidArgumentException